modules/ztp-acm-installing-disconnected-rhacm.adoc

```diff
@@ -17,10 +16,9 @@ You use {rh-rhacm-first} on a hub cluster in the disconnected environment to man
 +
 [NOTE]
 ====
-If you want to deploy Operators to the spoke clusters, you must also add them to this registry.
-See link:https://docs.openshift.com/container-platform/4.9/operators/admin/olm-restricted-networks.html#olm-mirror-catalog_olm-restricted-networks[Mirroring an Operator catalog] for more information.
+If you want to deploy Operators to the managed clusters, you must also add them to the disconnected registry. See xref:../../operators/admin/olm-restricted-networks.adoc#olm-mirror-catalog_olm-restricted-networks[Mirroring an Operator catalog] for more information.
```

Comment:
Cross references are not supported in module files. Links to the OCP docs from OCP modules are also not supported. You could update this to be an italicised reference to the heading instead.